nuxeo-jsf/nuxeo-platform-ui-web/src/main/java/org/nuxeo/ecm/platform/ui/web/tag/fn/DocumentModelFunctions.java

```diff
@@ -152,12 +122,11 @@ private static String getDefaultView(DocumentModel doc) {
     }
 
     private static LifeCycleService geLifeCycleService() {
```

Comment:
I think it's safe to remove this method - as the other ones - and use directly `Framework.getService(LifeCycleService.class)` where it is called.

Reply:
👍 yes, it already done by https://jira.nuxeo.com/browse/NXP-22532

Reply:
Are you sure? I think that https://jira.nuxeo.com/browse/NXP-22532 only removed the usage of `NXCore`. What I'm saying here is that you could remove the (badly named by the way) `geLifeCycleService` method and use `Framework.getService(LifeCycleService.class)` instead, just like you removed `getDirectoryService`, `getMimetypeService` and `getTypeManager`, to be consistent,

Reply:
@ataillefer you right, sorry i mixed the two. Yes i let this method because they log with error message when they don't find the service. if we don't care about having the message i can do it.

Reply:
Yes I believe that we don't care about the logs, if the service isn't loaded then the runtime will probably have crashed earlier.

Reply:
👍